packages/flame/lib/src/components/sprite_animation_group_component.dart

```diff
@@ -147,11 +167,20 @@ class SpriteAnimationGroupComponent<T> extends PositionComponent
   /// [autoResize] is true.
   void _resizeToSprite() {
     if (_autoResize) {
+      _isAutoResizing = true;
       if (animation != null) {
         size.setFrom(animation!.getSprite().srcSize);
       } else {
         size.setZero();
       }
+      _isAutoResizing = true;
```

Comment:
I made a mistake here. Should have been `_isAutoResizing = false;`. I have fixed this in #2457, but it is a big PR and might take some time for review. 

@spydon @erickzanardo, if another patch release planned, I can submit a separate PR for this.

Reply:
If you can move it out to a separate PR it would be great (and I can do another release later today), since the other PR would be a minor bump and not a patch bump.